test/unit/llm-events/langchain/vector-search-result.test.js

```diff
@@ -52,12 +56,19 @@ tap.beforeEach((t) => {
 })
 
 tap.test('create entity', async (t) => {
-  const search = new LangChainVectorSearchResult({
+  const search = new LangChainVectorSearch({
```

Comment:
this is fine but unnecessary. you just have to pass in some search_id. doesn't have to be from a LlmVectorSearch event